kythe/cxx/indexer/cxx/marked_source.cc

```diff
@@ -695,7 +695,7 @@ void MarkedSourceGenerator::ReplaceMarkedSourceWithTemplateArgumentList(
     std::string pre_text;
     {
       llvm::raw_string_ostream stream(pre_text);
-      print_arg.print(policy, stream);
```

Comment:
This requires a new version of LLVM

Reply:
ahh makes sense. I'll close this PR and we can fix it when we upgrade LLVM